types/mac.go

```diff
@@ -147,7 +143,12 @@ func (a *MACArray) Value(i int) net.HardwareAddr {
 	if a.IsNull(i) {
 		return nil
 	}
-	return net.HardwareAddr(a.Storage().(*array.Binary).Value(i))
+	b := a.Storage().(*array.Binary).Value(i)
+	if len(b) == 0 {
+		return make(net.HardwareAddr, 6)
```

Comment:
What is the purpose of `6` here or where is it coming from?

Reply:
minimal proper hardware addr (it was the same in append empty value), that will result in `"00:00:00:00:00:00"`

Reply:
Added const in 131cc3760eaf70d0cc9ea49a0495bead200959f4